test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -72,6 +72,7 @@ Could not find tests for procedure sys.printarg
 Could not find tests for procedure sys.sp_cursor_list
 Could not find tests for procedure sys.sp_describe_cursor
 Could not find tests for table sys.babelfish_helpcollation
+Could not find tests for table sys.babelfish_schema_permissions
```

Comment:
Can we add tests for this?

Reply:
No, we have not granted PUBLIC access to this catalog table since it is only meant for internal use.